src/components/PostInfo/PostInfo.tsx

```diff
@@ -1,5 +1,31 @@
 import React from 'react';
+import './PostInfo.scss';
+import { UserInfo } from '../UserInfo';
+import { CommentList } from '../CommentList';
+import { Post } from '../../types/Post';
 
-export const PostInfo: React.FC = () => (
-  <>Put the post here</>
+type Props = {
+  post: Post;
+};
+
+export const PostInfo: React.FC<Props> = ({ post }) => (
+  <div className="PostInfo">
+    <div className="PostInfo__header">
+      <h3 className="PostInfo__title">{post.title}</h3>
```

Comment:
same here, use destructuring of post object

Reply:
destructuring done.